src/util/fs-normalized.js

```diff
@@ -38,7 +38,7 @@ export const unlink: (path: string) => Promise<void> = promisify(require('rimraf
 export const copyFile = async function(data: CopyFileAction, cleanup: () => mixed): Promise<void> {
   try {
     await unlink(data.dest);
-    await copyFilePoly(data.src, data.dest, 0, data);
+    await copyFilePoly(data.src, data.dest, (fs.constants: any).COPYFILE_FICLONE || 0, data);
```

Comment:
Hmm. This is unfortunate.

The flow error you're suppressing here is because Flow's built-in types for Node.js are missing this constant. I have [submitted a PR to Flow](https://github.com/facebook/flow/pull/6791) to fix this, but even if that gets merged and released we can't immediately pull it in here. There were [Flow errors introduced with a recent update](https://github.com/yarnpkg/yarn/pull/5486) that still need to be fixed first.

So in the meantime, bypassing this type check in some manner is appropriate I think. Typically this is done with a `$FlowFixMe` comment though, so that we know that we should come back and fix this at some point. I don't know what the maintainers would prefer here.